tests/Speckle.Objects.Tests.Unit/Geometry/MeshTests.cs

```diff
@@ -6,22 +6,7 @@ namespace Speckle.Objects.Tests.Unit.Geometry;
 
 public class MeshTests
```

Comment:
Would be nice to have a test for mesh faces, to validate that ngon vertex counts add up to the full expected count